robot-server/robot_server/service/legacy/models/modules.py

```diff
@@ -249,7 +261,12 @@ class Config:
                                 "model": "heater-shaker_v10",
                                 "moduleModel": "heaterShakerModuleV1",
                                 "port": "/dev/ot_module_heatershaker1",
-                                "usbPort": {"hub": None, "port": None},
+                                "usbPort": {
+                                    "hub": None,
+                                    "port": None,
+                                    "port_group": None,
+                                    "hub_port": None,
+                                },
```

Comment:
We want to make sure the example structure is valid acc to the defined models-

```suggestion
                                "usbPort": {
                                    "hub": 1,
                                    "port": 1,
                                    "portGroup": 1,
                                    "hubPort": None,
                                },
```
If we change port group to have string values, then will need to update that here too.

Reply:
Also, I think the "port" field above "usbPort" might have been added here by mistake. We should remove it since a real response can't contain both 'port' (virtual port) and a 'usbPort'.